src/components/SettingsModal/index.jsx

```diff
@@ -167,6 +174,13 @@ export default class SettingsModal extends React.Component {
               >
                 Save Changes
               </Button>
+              <Button
+                variant="secondary"
+                onClick={this.handleDefault}
+                type="reset"
+              >
+                Reset Defaults
+              </Button>
```

Comment:
I lean more towards option 1... What about something like this:

![image](https://user-images.githubusercontent.com/2659980/110946212-9ce48400-830c-11eb-9b1b-0afae1d8513d.png)

Reply:
I also thought that some kind of feedback from `clear` for InVEST Jobs would be nice. Like a green check mark to the right of the button, or some kind of indication the action was taken. But maybe that's separate from this PR?

Reply:
Feedback for "Clear: sounds like a great idea. Your call on whether to do it in this PR or not. 

I think this is fine. The placement of Cancel and Save still make me assume they apply to everything here including the Clear jobs. But it's low stakes because if I came here to "Clear" it doesn't matter if I choose Cancel or Save on my way out.

Reply:
Yea, I actually still think it works okay. I feel like I've seen this in practice with `delete` rows for clearing cache or something and then having a big old `save` at the bottom for other things you might have changed. Google might do this for their gmail settings... But I'll go with the above layout for now!

Reply:
Update version here: https://github.com/natcap/invest-workbench/pull/132/commits/3a97cfdf5221f8366daf24cef318315e9d3ee08c